src/ResourceManager/Resources/Commands.Resources.Test/ScenarioTests/RoleAssignmentTests.cs

```diff
@@ -116,6 +117,9 @@ public void RaUserPermissions()
                                         .ResourceGroups
                                         .First();
 
+                    // Wait to allow newly created object changes to propagate
+                    Thread.Sleep(TimeSpan.FromSeconds(20));
```

Comment:
Please use this code instead https://github.com/Azure/azure-powershell/blob/dev/src/Common/Commands.Common/TestMockSupport.cs